packages/transpiler/src/transformations/transform-parents.ts

```diff
@@ -0,0 +1,26 @@
+import { getNodeSources } from '../solc/ast-utils';
+import { ContractDefinition } from '../solc/ast-node';
+import { Transformation } from '../transformation';
+
+export function transformParents(
```

Comment:
From what I understand, this transformation only renames the parent names from the inheritance specification (ie from `contract Foo is A, B => contract Foo is AUpgradeable, BUpgradeable`. Shouldn't we also change these names all through the source code, when calling a super function?